policyengine_us/variables/gov/states/sc/tax/income/deductions/net_capital_gain/sc_net_capital_gain_deduction.py

```diff
@@ -18,5 +18,11 @@ def formula(tax_unit, period, parameters):
         p = parameters(
             period
         ).gov.states.sc.tax.income.deductions.net_capital_gain
-        capital_gains = tax_unit("net_capital_gain", period)
-        return capital_gains * p.rate
+        qualified_dividends = add(
+            tax_unit, period, ["qualified_dividend_income"]
+        )
+        ltcg = add(tax_unit, period, ["long_term_capital_gains"])
+        stcg = add(tax_unit, period, ["short_term_capital_gains"])
+        net_capital_gain = max_(0, ltcg + stcg + qualified_dividends)
```

Comment:
replacing the above, though more importantly this doesn't seem to align with https://github.com/PolicyEngine/policyengine-us/issues/3452#issuecomment-1857948935 am i wrong?
```suggestion
        net_capital_gain = max_(0, add(tax_unit, period, ["qualified_dividend_income", "long_term_capital_gains", "short_term_capital_gains"]))
```

Reply:
I think we should include the `qualified_dividend_income` as part of the base. For the `short_term_capital_gains`, the example in issue 3452 shows that `short_term_capital_gains` can be negative, so I simply add them up.

Reply:
removed the dividend income and added the integration test from the form

Reply:
I have a quick question, why do we set the short term capital gains higher than 0, isn't the example shows that it can be negative?